hazenlib/tasks/acr_uniformity.py

```diff
@@ -78,73 +78,53 @@ def get_integral_uniformity(self, dcm):
         # Required pixel radius to produce ~1cm2 ROI
         r_small = np.ceil(np.sqrt(100 / np.pi) / self.ACR_obj.dx).astype(int)
         # Offset distance for rectangular void at top of phantom
-        d_void = np.ceil(5 / self.ACR_obj.dx).astype(int)
+        d_void = np.ceil(5 / self.ACR_obj.dy).astype(int)
         dims = img.shape  # Dimensions of image
 
-        (centre_x, centre_y), _ = self.ACR_obj.find_phantom_center(
-            img, self.ACR_obj.dx, self.ACR_obj.dy
-        )
+        mask=self.ACR_obj.get_mask_image(img)
+        masked_img=mask * img
```

Comment:
masked_img is not used. Is it supposed to be?

Reply:
I've now deleted all unused variables